src/components/common/PlainDraggable.vue

```diff
@@ -35,11 +35,29 @@ export class PlainDraggable extends mixins(Lang) {
     console.debug('PlainDraggable.vue', 'mounted')
 
     // this.$nextTick(() => {
+
+    this.draggable = new PlainDraggableLib(this.$el, this.plainDraggableLibOptions)
+    // draggable.rect.{left,top,x,y,...}
+
+    // todo: why this doesn't work?
+
+    // this.draggable.snap = {x: 50, y: 50, width: 50, height: 50}
+    // this.draggable.snap = {step: 21}
+
+    this.handleInitialized()
+    // })
+  }
+
+  updated(): void {
+    this.draggable.setOptions(this.plainDraggableLibOptions)
+  }
```

Comment:
`<Block>` passes updated coordinates to `<PlainDraggable startX startY>` which allows us to update block position when needed.